sdk/textanalytics/ai-text-analytics/review/ai-text-analytics.api.md

```diff
@@ -217,20 +208,29 @@ export interface ExtractKeyPhrasesSuccessResult extends TextAnalyticsSuccessResu
     keyPhrases: string[];
 }
 
-// @public (undocumented)
-export interface HealthcareAssertion {
-    association?: Association;
-    certainty?: Certainty;
-    conditionality?: Conditionality;
-}
-
 // @public
 export interface HealthcareEntity extends Entity {
-    assertion?: HealthcareAssertion;
+    assertion?: HealthcareEntityAssertion;
     dataSources: EntityDataSource[];
     normalizedText?: string;
 }
 
+// @public (undocumented)
+export interface HealthcareEntityAssertion {
+    association?: HealthcareEntityAssociation;
```

Comment:
`EntityAssociation` should be good enough, since it is under the `HealthcareEntityAssertion`. That logic also applies to  `EntityCertainty` and `EntityConditionality`

Reply:
We already use the `Healthcare` prefix for many types, e.g.:
  HealthcareEntityRelation,
  HealthcareEntityRelationRole,
  HealthcareEntityRelationRoleType

So I think we should be consistent and use the prefix `Healthcare`.

Reply:
cc @maririos @suhas92 @iscai-msft @abhahn

Reply:
I don't necessarily agree. The relations, roles, and types, are in fact specific to Healthcare.
The association, certainty, and conditionally could apply to a general entity. I understand it is not the case now, but it will be good to ask the service team if this concept could ever be applied to other entities/endpoints

Reply:
Ok I asked Ashly and he said there could be plans to add assertions for other type of entities in one or two semesters. Based on this, I will remove the Healthcare prefix.

Reply:
Thanks for the follow up Deya!